### ISSUE TITLE
Support multiple tensors per tensor page

### ISSUE THREAD

**Issue by ShadenSmith** [frostt-tensor/frostt-tensor.github.io#12]
Some data sources produce multiple tensors, such as:

  * [Chicago crimes](http://frostt.io/tensors/chicago-crime/)
  * [Reddit](http://frostt.io/tensors/reddit-2015/) (lots of options)
  * [Delicious](http://frostt.io/tensors/delicious/)
  * [Matrix multiplication](http://frostt.io/tensors/matmul/)

However, our [tensors page](http://frostt.io/tensors/) only supports one tensor per data source. I propose we instead support a list of tensor datasets and to list them all on the collections page. This considers the files in the `_tensors/` directory to be more like *data sources* which emit one of more *tensors*.

This will require some infrastructure work.

**Comment by fruitfly1026** [comment]
Yes, agree. Tensors generated from the same data source are quite interesting. We might need some different sub-names like "xx-[subname]" to distinguish them. I also can help with it. We can discuss more.
